conf/vhosts/vhosts.conf

```diff
@@ -11,29 +11,40 @@ server {
 
     location /api/ {
 
-        # When you need test the QPS of ngxway, you should remove the limit_traffic module.
-        # access_by_lua_block {
-        #    loadfile('/dist/ngxway/limiter/limit_traffic.lua')()
-        #    loadfile('/dist/ngxway/auth/check_sign.lua')()
-        # }
-        access_by_lua_file /dist/ngxway/main.lua; # access_by_lua_file can only load one lua file
-
-        # 启用缓存
+        # Define the cache of proxy.
         proxy_cache my_cache;
-        proxy_cache_lock on;
+        proxy_cache_lock off;
         proxy_cache_lock_timeout 5s;
 
+        # Define cache key and expiration.
```

Comment:
Must add this proxy_cache, even if you have cache controlled in the main.lua file